main.js

```diff
@@ -239,6 +239,12 @@ ipc.on('settings-changed', () => {
 // Update the themes of all open rulers on default theme change
 ipc.on('apply-default-theme', (evt, data) => {
 	rulers.forEach(ruler => ruler.send('update-theme', {filename: data.filename}));
+
+	// Focus all rulers temporarily to apply the changes.
+	// See: https://github.com/mikaa123/linear/issues/27
+	setTimeout(() => {
+		rulers.forEach(ruler => ruler.focus());
+	}, 300);
```

Comment:
we could, but in my tests I would only get consistent behaviors from electron if I added in a slight delay. Electron seems to have some timing problems with their events, this is actually why I left the link to the issue in the comment of the code I added in this PR, so that we can still close issue #27, but have something in the source code to remind us (or others) that is still something we can pay close attention to and optimize/change in the future. 

It's not as clean, but I say it would be best to keep the `300ms` delay just for a consistent user experience, and then revisit the issue of optimizing or changing the code when either"

- 1) we learn more about the Electron internals getting in the way of this feature or:
-  2) we wait for Electron to resolve some of the event timing behind the scenes and the problem goes away with an update.